init.js

```diff
@@ -13,7 +13,7 @@ if (process.env.NODE_ENV === 'production') {
 
 throng({ workers: 1, lifetime: Infinity }, () => {
 	const loadServer = () => {
-		return require('./server/server.js').startServer();
+		return require('./server/server').startServer();
```

Comment:
Relevant change. Note we're still executing the file directly from `server/` since this branch is still using `@babel/register`.